build/common.sh

```diff
@@ -490,16 +490,19 @@ function kube::build::build_image_cross() {
   local -r build_context_dir="${LOCAL_OUTPUT_ROOT}/images/${KUBE_BUILD_IMAGE}/cross"
   mkdir -p "${build_context_dir}"
   cp build/build-image/cross/Dockerfile ${build_context_dir}/Dockerfile
-  kube::build::docker_build "${KUBE_BUILD_IMAGE_CROSS}" "${build_context_dir}"
+  kube::build::docker_build "${KUBE_BUILD_IMAGE_CROSS}" "${build_context_dir}" 'true'
```

Comment:
> why single quotes?

Double-quotes indicates substitution intended, single-quotes indicates literal.

> and is there a reason to leave the one on L483 as false?

Yes, if we try to force-pull image that only exists locally (i.e. the image that the one on L483 is built on) Docker complains that it can't pull the image.  I will add a note.  Or maybe I should default to true?